opensanctions/crawlers/md_interdictie.py

```diff
@@ -0,0 +1,241 @@
+MONTHS = {
+    "ianuarie": "January",
+    "februarie": "February",
+    "martie": "March",
+    "aprilie": "April",
+    "mai": "May",
+    "iunie": "June",
+    "iulie": "July",
+    "august": "August",
+    "septembrie": "September",
+    "octombrie": "October",
+    "noiembrie": "November",
+    "decembrie": "December",
+}
+
+IDX_ORG_NAME = 1
+IDX_ORG_ADDRESS = 2
+IDX_ADMINS_FOUNDERS = 3
+IDX_APPLICANT = 4
+IDX_DECISION_NUM_DATE = 5
+IDX_REASON = 6
+IDX_REGISTRATION_DATE = 7
+IDX_DELAY_UNTIL = 8
+IDX_END_DATE = 9
+
+COUNTRY = "md"
+
+# Fondator/Administrator – 
+# or
+# Fondator - TOLMAŢCHI VALERI Administrator - TOLMAŢCHI VALERI
+# or
+# Lista conducătorilor - Galin Anatolie, Lista fondatorilor - Galin Anatolie
+# (List of leaders)
+# or
+# Conducători -
+# (Leaders)
+ROLES = { 
+    "Fondator/Administrator": "UNKNOWN",
+    "conducătorilor": "ADMINISTRATORS",
+    "Conducători": "ADMINISTRATORS",
+    "Fondator": "OWNERS",
+    "fondatorilor": "OWNERS",
+    "Administrator": "ADMINISTRATORS",
+}
+
+def crawl(context: Context):
+    path = context.fetch_resource("source.html", context.source.data.url)
+    context.export_resource(path, HTML, title=context.SOURCE_TITLE)
+    with open(path, "r") as fh:
+        doc = html.parse(fh)
+
+    table = doc.find(".//table")
+    headers = None
+    for row in table.findall(".//tr"):
+        if headers is None:
+            headers = [c.text_content() for c in row.findall("./th")]
+
+            # Assert that column order is as expected:
+            # fail explicitly upon possible breaking change
+            assert "Denumirea şi forma" in headers[IDX_ORG_NAME]
+            assert "Adresa  şi datele" in headers[IDX_ORG_ADDRESS]
+            assert "administratotul si fondatorii" in headers[IDX_ADMINS_FOUNDERS]
+            assert "Solicitantul" in headers[IDX_APPLICANT]
+            assert "Nr şi data deciziei" in headers[IDX_DECISION_NUM_DATE]
+            assert "temeiului de includere" in headers[IDX_REASON]
+            assert "Data înscrierii" in headers[IDX_REGISTRATION_DATE]
+            assert "Mențiuni" in headers[IDX_DELAY_UNTIL]
+            assert "Termenul limită" in headers[IDX_END_DATE]
+
+            continue
+
+        cells = row.findall("./td")
+        
+        name = cells[IDX_ORG_NAME].text_content()
+        
+        entity = context.make("Company")
+        entity.id = context.make_id(name, COUNTRY)
+        entity.add("name", name)
+        
+        address = parse_address(context, cells[IDX_ORG_ADDRESS].text_content())
+        h.apply_address(context, entity, address)
+
+        delay_until_date = parse_delay(context, cells[IDX_DELAY_UNTIL].text_content().strip())
+        start_date = delay_until_date or parse_date(cells[IDX_REGISTRATION_DATE].text_content().strip())
+        
+        sanction_num, decision_date = parse_sanction_decision(context, cells[IDX_DECISION_NUM_DATE].text_content().strip())
+        sanction = h.make_sanction(context, entity, sanction_num)
```

Comment:
use appropriate topic instead of sanction. Technically this isn't a sanction in international terms. It's just banned from public contracting.

Reply:
was adding the debarment topic to the entity like we do with other instances of "debarment" sufficient? I thought I'd be replacing the sanction but I see the other debarments have a "sanction" too.